sdks/java/io/rrio/src/test/java/org/apache/beam/io/requestresponse/EchoITOptions.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.io.requestresponse;
+
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.testinfra.mockapis.echo.v1.EchoServiceGrpc;
+
+/**
+ * Shared options for running integration tests on a deployed {@link EchoServiceGrpc}. See <a
+ * href="https://github.com/apache/beam/tree/master/.test-infra/mock-apis#integration">https://github.com/apache/beam/tree/master/.test-infra/mock-apis#integration</a>
+ * for details on how to acquire values required by {@link EchoITOptions}.
+ *
+ * <p>To provide these values to your integration tests:
+ *
+ * <pre>
+ *   ./gradlew :sdks:java:io:rrio:integrationTest -DintegrationTestPipelineOptions='[
+ *      "--grpcEndpointAddress=",
+ *      "--httpEndpointAddress="
+ *   ]'
+ * </pre>
+ */
+public interface EchoITOptions extends PipelineOptions {
+  @Description("The gRPC address of the Echo API endpoint, typically of the form <host>:<port>.")
+  String getgRPCEndpointAddress();
```

Comment:
Can you test this explicitly to make sure this works as intended?

It feels to me that it should be `getGRPCEndpointAddress` to follow Java conventions, but I am not sure if the PipelineOptions is case sensitive.

Reply:
I experimented with `getGRPCEndpointAddress` and it is case sensitive.  Setting as `getgRPCEndpointAddress` allows the flag to look like this: `--gRPCEndpointAddress`. Using `getGRPCEndpointAddress` looks like this: `--GRPCEndpointAddress`. Given the canonical name is with the little 'g', I opted for `getgRPCEndpointAddress`.

Reply:
I will go ahead and resolve this but please re-open if you feel more discussion is warranted. Thank you, again for reviewing!